readme.md

```diff
@@ -200,104 +107,224 @@ $ adb forward tcp:9222 localabstract:chrome_devtools_remote
 $ lighthouse --disable-device-emulation --disable-cpu-throttling https://mysite.com
 ```
 
-## Tests
+## Using programmatically
 
-Some basic unit tests forked are in `/test` and run via mocha. ESLint also checks for style violations.
+The example below shows how to setup and run Lighthouse programmatically as a Node module.
```

Comment:
Open sourcing a more convenient launcher would be great.  That's really the bulk of the example, and something I struggled to get working for my demo.

When looking into a lib, I do really like seeing install + different usages in the main readme. Since a lot of people as for docs on this, I'd love to keep it front and center. Ironically, and email thread from Addy prompted me to add this...which led to a full reorg :)

Happy to move it out though.